assignments/chess-board/code.js

```diff
@@ -38,11 +38,21 @@ let board =
 const row = (y) => Math.floor(y / (height / 8))
 const column = (x) => Math.floor(x / (width / 8))
 
-const select = (x, y) => {
-  drawText(board[row(y)][column(x)], width / 8 * column(x), height / 8 * (row(y) + 1), 'red', 45)
+let aSelect = []
+const move = (x, y) => {
+  if (aSelect !== []) {
+    drawText(aSelect, width / 8 * column(x), height / 8 * (row(y) + 1), 'black', 30)
+    board[row(y)][column(x)] = aSelect[0]
+    aSelect.pop(board[row(y)][column(x)])
+  } else {
+    drawText(board[row(y)][column(x)], width / 8 * column(x), height / 8 * (row(y) + 1), 'red', 40)
+    board[row(y)][column(x)] = ''
+    aSelect.push(board[row(y)][column(x)])
```

Comment:
And swap these and change the `aSelect` line to:

```suggestion
    selectedPiece = board[row(y)][column(x)]
    board[row(y)][column(x)] = ''
```